polaris-react/src/components/Text/Text.tsx

```diff
@@ -63,6 +63,8 @@ export interface TextProps {
   fontWeight?: FontWeight;
   /** HTML id attribute */
   id?: string;
+  /** Render OpenType font characters with consistent height and width */
+  monospace?: boolean;
```

Comment:
Thanks @aveline. That's an interesting point. I understand the potential for confusion and reasoning behind changing the prop name. My only reservation in making the change is the expectation that the term "monospace" will be used to describe the requirement, as it was on the [ticket to introduce the formatting](https://github.com/Shopify/shopify/issues/407753). I think the benefit of the semantic consistency of the prop name to the way it's described might outweigh any potential confusion. @alex-page what are your thoughts?

Reply:
I think I would call this `numeric` as it's applying `font-variant-numeric` with the values we recommend.

Reply:
@alex-page - I'm late to this party, apologies. Why are we using `font-variant-numeric` rather than `font-variant-mono`. I don't see the numeric variant in our [font tokens](https://polaris.shopify.com/tokens/font) and I figure we should always try to back our components with available token values.